.gitpod.yml

```
@@ -29,13 +31,19 @@ vscode:
   extensions:
     - golang.go
 
-workspaceLocation:
-  mattermost-gitpod-config/mattermost.code-workspace
+workspaceLocation: mattermost-gitpod-config/mattermost.code-workspace
 
 tasks:
+  - name: Welcome
+    init: |
+      npm i -g markserv
+    command: |
+      markserv -p 9000 &
+      gp preview "$(gp url 9000)/welcome.md"
```

Since this is the only markdown file right now, 1/5 I think we should have `welcome.md` served directly. This gives utility to the `/` route on the markdown server. If we add another markdown file, we can change it to serve multiple files.

```suggestion
      markserv welcome.md -p 9000 &
      gp preview $(gp url 9000)
```

---

I tried this way, but sadly it would not load the file directly with this command. It would just open the directory still. When your Gitpod is working again, you should try on your end and see if you can get that specific command to work as expected.

---

I agree with this idea of serving the welcome page directly since it's the only page currently, if it's technologically possible.